lib/config-generator.js

```diff
@@ -148,11 +148,25 @@ class ConfigGenerator {
             entry[sharedEntryTmpName] = tmpFilename;
         }
 
-        if (this.webpackConfig.copyFilesConfigs.length > 0) {
+        const copyFilesConfigs = this.webpackConfig.copyFilesConfigs.filter(entry => {
+            const copyFrom = path.resolve(
+                this.webpackConfig.getContext(),
+                entry.from
+            );
+
+            if (!fs.existsSync(copyFrom) || !fs.lstatSync(copyFrom).isDirectory()) {
+                logger.warning(`The "from" option of copyFiles() should be set to an existing directory but "${entry.from}" does not seem to be one. No file will be copied for this entry.`);
```

Comment:
"entry" might not be the best word here... because it means something else

Reply:
Hmmm, maybe... by "entry" I meant "config entry", would it be better to put that?